app/models/keyword.rb

```diff
@@ -11,4 +11,10 @@ class Keyword < ApplicationRecord
             presence: true,
             numericality: { greater_than_or_equal_to: 0 },
             if: :succeeded?
+
+  after_create_commit :enqueue_scrape_job
```

Comment:
We are also missing the unit test for this part - after creating a keyword, it enqueues the job

Reply:
Yep, I left a note on this PR. Once the #31 is merged, this test will be implemented

Reply:
Next time instead of waiting for the #31 is merged, you can checkout from there directly to have the full flow, what do you think about that idea?

Reply:
wouldn't it make make code reviewing much harder, since we can not isolate changes from this PR 🤔 ?

Reply:
Added test in 32de9e2b3f09716e7fb500a9a6bf522263a955b7